src/pcp2json/pcp2json.py

```diff
@@ -47,6 +48,7 @@
 CONFVER = 1
 INDENT = 2
 TIMEFMT = "%Y-%m-%d %H:%M:%S"
+TIMEOUT = 2.5 # seconds
```

Comment:
Is this something that doesn't need to be configured? If so, would slightly larger timeout be better?

Reply:
Oh - I just did the same thing pcp2influxdb does by hardcoding this.  But I think this use-case is more likely to send to a faraway host so I've added a new config option now.